src/data_provider/src/osinfo/sysOsParsers.cpp

```diff
@@ -137,15 +137,28 @@ static bool findVersionInStream(std::istream& in,
 bool UnixOsParser::parseFile(std::istream& in, nlohmann::json& info)
 {
     constexpr auto SEPARATOR{'='};
-    static const std::map<std::string, std::string> KEY_MAP
+    bool ret;
+
+    static const std::map<std::string, std::string> KEY_MAP1
     {
         {"NAME",             "os_name"},
         {"VERSION",          "os_version"},
         {"ID",               "os_platform"},
         {"BUILD_ID",         "os_build"},
         {"VERSION_CODENAME", "os_codename"}
     };
-    const auto ret {parseUnixFile(KEY_MAP, SEPARATOR, in, info)};
+    ret = parseUnixFile(KEY_MAP1, SEPARATOR, in, info);
+
+    if (info.find("os_version") == info.end())
+    {
+        static const std::map<std::string, std::string> KEY_MAP2
+        {
+            {"VERSION_ID",       "os_version"},
+        };
+        in.clear();
+        in.seekg(0);
+        ret |= parseUnixFile(KEY_MAP2, SEPARATOR, in, info);
+    }
```

Comment:
This change is unnecesary if you add new key(VERSION_ID) in the previous static map.

Reply:
done